sources/packages/web/src/components/institutions/modals/AddInstitutionUserModal.vue

```diff
@@ -39,6 +39,17 @@
               >
               <InputSwitch v-model="isAdmin" />
             </v-col>
+            <v-col v-if="isAdmin">
+              <span class="form-text text-muted mb-2"
+                ><strong>Select Admin Role</strong></span
```

Comment:
maybe you can use bootstrap class to achieve bold, instead of `<strong>`. https://getbootstrap.com/docs/5.0/utilities/text/, https://getbootstrap.com/docs/4.0/utilities/text/,

Reply:
bootstrap class font-weight-bold used.